pandas/tests/indexes/timedeltas/test_indexing.py

```diff
@@ -11,6 +11,17 @@
 class TestTimedeltaIndex(object):
     _multiprocess_can_split_ = True
 
+    def test_contains(self):
+        # Checking for any NaT-like objects
+        # GH 13603
+        td = pd.to_timedelta(range(5), unit='d') + pd.offsets.Hour(1)
+        for v in [pd.NaT, None, float('nan'), np.nan]:
```

Comment:
maybe should consolidate these types of tests in scalar/test_nat.py (even though you are testing the index itself here).